wfapi/error.go

```diff
@@ -244,30 +240,29 @@ func ErrorMissingCatalogEntry(ref CatalogRef, replayAvailable bool) error {
 	var msg string
 	var available string
 	if replayAvailable {
-		msg = fmt.Sprintf("catalog entry %q exists, but content is missing. Re-run recusively to resolve entry.", ref.String())
+		msg = "catalog entry {{catalogRef | q}} exists, but content is missing. Re-run recusively to resolve entry."
 		available = "true"
 	} else {
-		msg = fmt.Sprintf("missing catalog entry %q", ref.String())
+		msg = "missing catalog entry {{catalogRef | q}}"
 		available = "false"
 	}
 	return serum.Error(ECodeCatalogMissingEntry,
-		serum.WithMessageLiteral(msg),
+		serum.WithMessageTemplate(msg),
 		serum.WithDetail("catalogRef", ref.String()),
 		serum.WithDetail("replayAvailable", available),
 	)
 }
 
+// DEPRECATED: adds no value
```

Comment:
I think maybe this could be more helpful if it says what one should do instead.

`ECodeGit` is considered useful maybe?  And we want to say "just use serum.Error; there is no value-add from an additional constructor func"?  Is that what is meant?

Reply:
(Same goes for other occurrences of this comment further down.)

Reply:
>Is that what is meant?

Yes.

Reply:
Deprecation comments are now more descriptive.